security/README.md

```diff
@@ -0,0 +1,57 @@
+Security
+========
+
+A guide for practicing safe web.
+
+Think
+-----
+
+Security is important, and you can't practice these guidelines without
+understanding them. Make sure you understand each guideline, why it exists, and
+how to follow it.
+
+Failing to follow these guidelines will likely put you, your team, and your
+deployed services at risk of compromise or loss of privacy.
+
+Using Passwords
```

Comment:
Oh really sorry: I meant that this section is more of a set of best practices, whereas the others are basically mandatory.

My wording ("that first section") was confusing and vague and I'm really sorry about that.